07-routing-gossip.md

```diff
@@ -359,6 +384,12 @@ to be ordered in ascending order, unknown ones can be safely ignored.
 Additional fields beyond `addresses` may also be added in the future—with
 optional padding within `addresses`, if they require certain alignment.
 
+If a node signals `option_will_fund`, they are signaling that they
+will provide funding to a node at the stated terms. They also commit
+to a feerate they will charge for transmitting funds over the channel
+for the duration of the funding lease.
+
+
 ### Security Considerations for Node Aliases
```

Comment:
Need a new requirement in `channel_update` that we don't send out feerates that violate any active lease.